### DISCUSSION TITLE
Plausible as a platform?

### DISCUSSION THREAD

**Discussion by HuynhBrian** [plausible/analytics#2425] [Anything Else]
Hello,

We're building an eCommerce platform to let people create online stores. We personally used Plausible and love it. How can we offer our users Plausible analytics for each stores that they created? If this isnt possible, what analytics platform would you recommend to achive this goal?

**Comment by metmarkosaric** [comment]
Thanks for reaching out @HuynhBrian! Sure, this type of integration is possible and we have several features that help with this use case:

- [Embed dashboard](https://plausible.io/docs/embed-dashboard) allows you to embed our default dashboard (without Plausible branding) in your UI. This is the easy and quick option.

- [Stats API](https://plausible.io/docs/stats-api) allows you to take individual metrics and create a custom-built dashboard within your UI in whichever shape or form you want. This requires more development time from your side but you get full flexibility.

- [Sites API](https://plausible.io/docs/sites-api) allows you to add and manage a large number of sites programmatically.

- [Proxy](https://plausible.io/docs/proxy/introduction) allows you to run Plausible as the first-party connection from your domain name so there are no third-party calls and you get more accurate stats as adblockers don't block it.

If you need some help or have any questions, do [reach out to our support](https://plausible.io/contact). Thanks!

**Reply by HuynhBrian** [reply]
Thank you for your insightful response @metmarkosaric! Seems like the Sites API might be it. Will dive into it.
